Signal/src/ViewControllers/BlockListUIUtils.h

```diff
@@ -18,32 +19,32 @@ typedef void (^BlockActionCompletionBlock)(BOOL isBlocked);
 
 #pragma mark - Block
 
-+ (void)showBlockContactActionSheet:(Contact *)contact
-                 fromViewController:(UIViewController *)fromViewController
-                    blockingManager:(OWSBlockingManager *)blockingManager
-                    contactsManager:(OWSContactsManager *)contactsManager
-                    completionBlock:(nullable BlockActionCompletionBlock)completionBlock;
```

Comment:
Our contact-related views should deal in `ContactAccount` (which specifies which recipientId should be used for contacts with more than one), not `Contact`.